components/support/rc_crypto/src/ece_crypto.rs

```diff
@@ -159,102 +163,27 @@ impl CryptoImpl {
         Ok(aead::open(&key, nonce, aead::Aad::empty(), &ciphertext)?)
     }
 
-    fn _random(dest: &mut [u8]) -> crate::Result<()> {
+    fn random(&self, dest: &mut [u8]) -> Result<(), ece::Error> {
```

Comment:
Naming nit, since you're changing this API anyway on the rust-ece side...I wonder if `random_bytes` or similar more explicit name might add a bit more clarity, given the other methods all seem to have longer and more explicit names.